service-commands/scripts/seed-audio.sh

```diff
@@ -0,0 +1,38 @@
+# Gets the spl-token wallet from the discovery node api
+get-user-bank () {
+    curl "http://dn1_web-server_1:5000/users?id=$1" &> /dev/null | jq .data\[0\].spl_wallet | sed 's/"//g'
+}
+
+# Gets the private key for a wallet out of the seed cache
+get-eth-private-key () {
+    jq "map(select(.userId==$1).wallet.privKey)[0]" < ~/.audius/seed-cache.json | sed 's/"//g'
```

Comment:
this will work but FYI, to get raw value out of jq without spaces or quotes you can do `jq -rc` (raw + compact flags) :)

Reply:
amazing! til